elements/pfe-tabs/src/pfe-tab-panel.scss

```
@@ -32,24 +38,18 @@ $LOCAL-VARIABLES: (focus: pfe-var(link),
   .container {
     margin: 0;
     width: 100%;
-    background-color: pfe-local(BackgroundColor, $region: panel);
-    border-top: pfe-local(BorderTop, $region: panel);
-    border-right: pfe-local(BorderRight, $region: panel);
-    border-bottom: pfe-local(BorderBottom, $region: panel);
-    border-left: pfe-local(BorderLeft, $region: panel);
-
-    padding: pfe-local(Padding, $region: panel); //16px
-
-    @media screen and (min-width: pfe-breakpoint(md)) {
-      padding: calc(#{pfe-local(Padding, $region: panel)} * 2);
-    }
-
-    @media screen and (min-width: pfe-breakpoint(lg)) {
-      padding: calc(#{pfe-local(Padding, $region: panel)} * 3);
-    }
+    background-color: pfe-local(BackgroundColor, $region: panel, $fallback: transparent);
+    border-top: pfe-local(BorderTop, $region: panel, $fallback: 0);
+    border-right: pfe-local(BorderRight, $region: panel, $fallback: 0);
+    border-bottom: pfe-local(BorderBottom, $region: panel, $fallback: 0);
+    border-left: pfe-local(BorderLeft, $region: panel, $fallback: 0);
+    padding-top: pfe-local(PaddingTop, $region: panel, $fallback: calc(#{pfe-var(container-spacer)} * 3));
```

```suggestion
    padding-top: pfe-local(PaddingTop, $region: panel);
```

Since the default for padding-top in the map above is also 48px, you won't need the fallback value here.